### PR TITLE
added actions and props for cycling through blue,brown,green,grey

### PR DESCRIPTION
**Pull request by acatmore**

### CODE REVIEW

File: src/Cell.js [hunk 1]
```diff
@@ -10,16 +10,25 @@ const StyledCell = styled.div`
...
+  height: 20px;
   cursor: pointer;
   user-select: none;
   background-color: ${props => props.cell.backgroundColor || "#eee"};
 `;
 
 export class Cell extends Component {
   handleClick() {
-    this.props.dispatch(turnBlue(this.props.x, this.props.y));
+    console.log("hi");
```
Comment (iameli) [inline src/Cell.js]: that's very friendly but not terribly necessary

File: src/actions.js [hunk 2]
```diff
@@ -8,10 +8,41 @@ export const initGrid = (width, height) => {
 };
 
 export const TURN_BLUE = "TURN_BLUE";
-export const turnBlue = (x, y) => {
+export const turnBlue = (x, y, clickedNum) => {
```
Comment (iameli) [inline src/actions.js]: I don't think `clickedNum` is necessary in the actions. Evidence: you're not actually providing the argument in these actions and everything works fine.

image

Seems like the reducer is taking care of it.

**Review by iameli — CHANGES_REQUESTED** [review]
Two changes, looks fine otherwise

File: src/actions.js (hunk 2 above)
Comment (acatmore) [reply src/actions.js]: I see. Was wondering about that. I had it set it up before I had the prop built in from the reducer

**Review by iameli — APPROVED** [review]

**Pull request merged**